arangod/Cluster/ClusterInfo.cpp

```diff
@@ -6293,14 +6293,14 @@ ClusterInfo::ServersKnown::ServersKnown(
   }
 }
 
-std::unordered_map<ServerID, ClusterInfo::ServersKnown::KnownServer> const&
+containers::FlatHashMap<ServerID, ClusterInfo::ServersKnown::KnownServer> const&
```

Comment:
It looks like this method is not called from anywhere, so we can as well remove it.

Reply:
I will try